stash_engine/spec/config/app_config.yml

```diff
@@ -11,7 +11,7 @@ test:
     service: stripe
     key: travis_test_key
     data_processing_charge: 12000 # charge in cents
-
+  merritt_express_base_url: http://dummy_domain.example.com
```

Comment:
be sure to add this to the Dryad repo's example as well

Reply:
Added.